attributes/default.rb

```diff
@@ -45,6 +47,17 @@
   zk_announce_path: '/aurora/svc'
 }
 
+default['aurora']['client'] = [
+  {
+    auth_mechanism: "UNAUTHENTICATED",
+    name: "test-cluster",
+    scheduler_zk_path: "/aurora/scheduler",
+    slave_root: "/var/lib/mesos",
+    slave_run_directory: "latest",
+    scheduler_uri: "http://localhost:8082",
+  }
+]
+
```

Comment:
Indeed, it looks like most options there are useless. IMO we don't need to use ZK from the client. I'll update this accordingly